test/compatibility/threading/Gemfile

```diff
@@ -1,8 +1,8 @@
-source :rubygems
+source 'https://rubygems.org'
 
 gem "pg"
 gem "mysql2"
 gem "mocha"
-gem "activerecord", "3.1.1"
+gem "activerecord", "3.2.22.1"
```

Comment:
Can we bump all the way to `3.2.22.5`, that's what monorail is on.

Reply:
Yep can do.